app/views/common/_pagination.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
+  <%=
+    per_page_options = [5, 10, 15, 25, 50]
+    per_page_options << Setting[:entries_per_page].to_i
+    per_page_options << params[:per_page].to_i unless params[:per_page].blank?
```

Comment:
actually, most do support it as they use the ```resource_base_search_and_page``` helper. i think some that do not should be fixed (in this pr or another pr before this one) - i find architectures, hosts and facts missing the parameter.